src/alert.cpp

```diff
@@ -245,8 +247,20 @@ bool CAlert::ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThre
     return true;
 }
 
-void
-CAlert::Notify(const std::string& strMessage, bool fThread)
+// static
```

Comment:
The comment is a habit I have to indicate that the method just below is static. Something that is in the signature in the header, but not possible to have in any other way in the cpp file.